src/actions/pause.js

```diff
@@ -57,12 +59,14 @@ export function resumed() {
       return;
     }
 
+    const wasPausedInEval = pausedInEval(getState());
+
     dispatch({
       type: "RESUME",
       value: undefined
     });
 
-    if (!isStepping(getState())) {
+    if (!isStepping(getState()) && !wasPausedInEval) {
```

Comment:
we check the `frame.displayName`, because the debugger is now resuming, it's no longer evaluating an expression